### PR TITLE
MONGOID-5275 Embedded Associations inside submodules cannot find inverses

### PR DESCRIPTION
**Pull request by Neilshweky**
MONGOID-5275

### CODE REVIEW

**Comment by johnnyshields** [comment]
@Neilshweky I'll be surprised is the fix is as simple as you have it here. I suspect we may need to rework the relation binding, but if it really is this simple, then great!

**Comment by Neilshweky** [comment]
> @Neilshweky I'll be surprised is the fix is as simple as you have it here. I suspect we may need to rework the relation binding, but if it really is this simple, then great!

Oh I don't think it is but I threw it up in two min just to see what would happen. I'll learn more about the problem from the errors I get.

File: lib/mongoid/association/embedded/embeds_many.rb [hunk 1]
```diff
@@ -168,9 +168,9 @@ def polymorphic_inverses(other = nil)
 
         def determine_inverses(other)
           matches = relation_class.relations.values.select do |rel|
+            assoc_class = rel.relation_class rescue next
```
Comment (Neilshweky) [inline lib/mongoid/association/embedded/embeds_many.rb:171]: I'm making an assumption here, that if the class doesn't exist, we should not consider this association as a candidate for an inverse.

**Review by comandeo — APPROVED** [review]

**Comment by p-mongo** [comment]
Is this waiting for the association target class epic?

**Comment by Neilshweky** [comment]
> Is this waiting for the association target class epic?

If you mean the class autoloading epic, yes.

**Pull request closed without merging**